ui/src/main/webapp/src/app/app.routing.ts

```diff
@@ -71,16 +71,15 @@ export const appRoutes: Routes = [
                 component: DefaultLayoutComponent,
                 children: [
                     {path: '', redirectTo: "/project-list", pathMatch: "full"},
+                    {path: 'about', component: AboutPageComponent},
                     {
                         path: "configuration",
-                        resolve: { configuration: ConfigurationResolve },
```

Comment:
@klinki - I think that he ran some sort of autoformatter on it. It definitely made the PR confusing, though, since we haven't used that autoformatter on the file before.